### PR TITLE
feat(coverage): add section on coverage

### PR DESCRIPTION
**Pull request by alexlrobertson**
Closes: #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +108,16 @@ contextService.getToolContext.should.have.been.calledOnce
     .and.calledWithExactly('color', 'Purple Mountains Majesty');
 ```
 
+## Coverage
+
+Code coverage helps to inform you what parts of a project are well tested and what parts are in need of testing. Code coverage not only indicates line by line, but each logic branch that is executed in a test.
+
+Code coverage should not be treated as a quality target, but a tool to discover untested code.
+
+It is important to actually tests all parts of a program, not just execute all parts.
```
Comment (tykowale) [inline README.md]: :100:

Comment (alexlrobertson) [inline README.md]: `to actually tests` -> `to actually test`

File: README.md [hunk 2]
```diff
@@ -107,6 +108,16 @@ contextService.getToolContext.should.have.been.calledOnce
     .and.calledWithExactly('color', 'Purple Mountains Majesty');
 ```
 
+## Coverage
+
+Code coverage helps to inform you what parts of a project are well tested and what parts are in need of testing. Code coverage not only indicates line by line, but each logic branch that is executed in a test.
```
Comment (tykowale) [inline README.md]: This sentence is a mouth full and a little confusing.

Comment (alexlrobertson) [inline README.md]: :eyes:

**Comment by tykowale** [comment]
:cake:

**Pull request merged**